### PR TITLE
Exclude options is not working

### PR DESCRIPTION
**Pull request by juri-t**
The Exclude option wasn't working, so I fixed it.

### CODE REVIEW

File: generator.go [hunk 1]
```diff
@@ -65,12 +65,16 @@ func (d *generator) Generate(opts *GenerateOptions) error {
 		if err := rows.Scan(&table); err != nil {
 			return err
 		}
+		exclude := false
 		for _, e := range opts.Exclude {
 			if e == table {
+				exclude = true
```
Comment (keroxp) [inline generator.go:71]: How about jumping after `append()` by `goto`?

Comment (juri-t) [reply generator.go]: I avoided using goto, but it looks good.
I fixed in d8b387d.
Label EOL stands for End of Loop.

**Review by keroxp — COMMENTED** [review]
@juri-t Looks fine. Would you add tests?

**Comment by juri-t** [comment]
@keroxp 
I will not add any tests. The interface is not suitable for unit testing, so I think it's enough to check the operation.

**Pull request merged**